.github/workflows/nodejs.yml

```diff
@@ -27,4 +27,6 @@ jobs:
         run: yarn build
 
       - name: Test
-        run: yarn test --silent
+        run: yarn test:ci --silent
+        env:
+          NODE_OPTIONS: '--max_old_space_size=4096'
```

Comment:
The memory issue is quite weird. I've been testing this locally and I couldn't notice any substantial increase in the memory usage when using Jest. It has also used "only" 600MB of the total memory heap while the limit on CI is 2GB so it shouldn't cause any issues. Apparently, it did though and increasing the limit to 4GB helps. 🤷‍♂️